styles/panel.css

```diff
@@ -0,0 +1,103 @@
+.panel {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+
+    width: 340px;
+    height: 432px;
+
+    grid-column: 2;
+    grid-row: 1 / 3;
+}
+
+.tab__btns-container {
+    margin-top: 2px;
+    display: flex;
+    justify-content: flex-start;
+    width: 100%;
+    margin-bottom: -2px;
```

Comment:
Consider other ways of solving this problem, for example using a pseudo-element (`::after`) to cover the gap.